### PR TITLE
Single local login

### PR DESCRIPTION
**Pull request by Neloop**

### CODE REVIEW

File: app/V1Module/presenters/RegistrationPresenter.php [hunk 1]
```diff
@@ -103,7 +103,7 @@ public function actionCreateAccount() {
 
     // check if the email is free
     $email = trim($req->getPost("email"));
-    if ($this->users->getByEmail($email) !== null) {
+    if ($this->logins->getByUsername($email) !== null) {
```
Comment (krulis-martin) [inline app/V1Module/presenters/RegistrationPresenter.php]: This is somewhat strange -- getByUsername($email). This method should have a better name, or an explanatory comment is required.

Comment (Neloop) [reply app/V1Module/presenters/RegistrationPresenter.php]: I do not follow... this is classical repository method which finds a login based on given username... what do you find strange about it?

Comment (krulis-martin) [reply app/V1Module/presenters/RegistrationPresenter.php]: When you say user name, first that comes to mind is firstName and lastName. getByLogin or getByLoginName would be much better.

Comment (Neloop) [reply app/V1Module/presenters/RegistrationPresenter.php]: But the column is `username`, therefore, repository convention says to have this method name...

Comment (Neloop) [reply app/V1Module/presenters/RegistrationPresenter.php]: And no... we are not gonna rename the database column...

Comment (krulis-martin) [reply app/V1Module/presenters/RegistrationPresenter.php]: OK, can you at least leave a note (in a comment) that username is in fact login?

Comment (Neloop) [reply app/V1Module/presenters/RegistrationPresenter.php]: Can do

**Pull request merged**